packages/core-components/src/layout/Sidebar/SidebarSubmenuItem.tsx

```diff
@@ -163,7 +166,7 @@ export const SidebarSubmenuItem = (props: SidebarSubmenuItemProps) => {
           <div className={classes.dropdown}>
             {dropdownItems.map((object, key) => (
               <Link
-                component={NavLink}
+                component={isExternalLink(object.to) ? BackstageLink : NavLink}
```

Comment:
Man, this makes me wonder if we're making it too complex.

If you look at [the implementation of our Link](https://github.com/backstage/backstage/blob/b2efb8d544cf5f97d3d36445e542e5a6f53dafd3/packages/core-components/src/components/Link/Link.tsx#L78), you'll see that it already does this entire dance of figuring out if it's external or not, and acting accordingly.

Instead of your change here, would it give the same result if you reverted this entire file, and just replaced the import at the top instead?

```diff
-import Link from '@material-ui/core/Link';
+import { Link } from '../../components/Link';
```

I.e. just using that Link in the first place, entirely.

Reply:
That's a good idea and definitely much cleaner